hopglass-server.js

```diff
@@ -563,11 +592,20 @@ function getMetrics(stream) {
       stream.write('meshnodes_total ' + counter_meshnodes_total + '\n')
       stream.write('meshnodes_online_total ' + counter_meshnodes_online_total + '\n')
       stream.write('total_clients ' + counter_clients + '\n')
-      stream.write('total_traffic_rx ' + counter_traffic_rx + '\n')
-      stream.write('total_traffic_mgmt_rx ' + counter_traffic_mgmt_rx + '\n')
-      stream.write('total_traffic_tx ' + counter_traffic_tx + '\n')
-      stream.write('total_traffic_mgmt_tx ' + counter_traffic_mgmt_tx + '\n')
-      stream.write('total_traffic_forward ' + counter_traffic_forward + '\n')
+
+      var labels = [];
```

Comment:
I wonder how this works. You can't have a String as index for an array.
You should be using an Object ({}) here. 
If you don't mind I will redo this.